src/main/java/com/rarchives/ripme/ripper/rippers/ChanRipper.java

```diff
@@ -19,7 +19,7 @@
 
 public class ChanRipper extends AbstractHTMLRipper {
     public static List<ChanSite> explicit_domains = Arrays.asList(
-        new ChanSite(Arrays.asList("boards.4chan.org"),   Arrays.asList("4cdn.org", "is.4chan.org")),
+        new ChanSite(Arrays.asList("boards.4chan.org"),   Arrays.asList("4cdn.org", "is.4chan.org", "is2.4chan.org")),
```

Comment:
Yeah, also I don't think this `explicit_domains` field is handled in terms of regex, just explicit strings, so that would require modifying. Again, not a blocking issue for this PR. I'll open an issue.